dicoogle/src/main/resources/webapp/js/components/indexer/TaskStatus.jsx

```diff
@@ -15,6 +15,7 @@ class TaskStatus extends React.Component {
         complete: PropTypes.bool,
         canceled: PropTypes.bool,
         taskProgress: PropTypes.number,
+        taskTimeCreated: PropTypes.number,
```

Comment:
It should then become a string with the proposed change.

```suggestion
        taskTimeCreated: PropTypes.string,
```

Reply:
Fixed.